### ISSUE TITLE
Goal of covering RFC 9535?

### ISSUE THREAD

**Issue by glyn** [jsonpath-standard/jsonpath-compliance-test-suite#61]
Should we update the README to indicate that we are working towards coverage of RFC 9535? The internet draft link might become confusing if and when v2 work kicks off.

**Comment by gregsdennis** [comment]
Yes, that seems appropriate.

**Comment by ahx** [comment]
Yes. I would like to understand what "version" (?) of JSONPath this is about. The link to https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base is confusing. A link to a published version (https://www.rfc-editor.org/rfc/rfc9535 ?) would be more helpful.
